bottlecap/src/metrics/enhanced/lambda.rs

```diff
@@ -573,15 +558,25 @@ impl Lambda {
 
         tokio::spawn(async move {
             // get list of all process ids
-            let pids = proc::get_pid_list();
+            let mut pids = proc::get_pid_list();
 
-            // Set fd_max and initial value for fd_use to -1
+            // Set fd_max and initial value for fd_use
             let fd_max = proc::get_fd_max_data(&pids);
-            let mut fd_use = -1_f64;
+            let mut fd_use = if let Ok(fd_use) = proc::get_fd_use_data(&pids) {
+                fd_use
+            } else {
+                debug!("Could not get file descriptor use.");
+                -1_f64
+            };
 
-            // Set threads_max and initial value for threads_use to -1
+            // Set threads_max and initial value for threads_use
             let threads_max = proc::get_threads_max_data(&pids);
-            let mut threads_use = -1_f64;
+            let mut threads_use = if let Ok(threads_use) = proc::get_threads_use_data(&pids) {
+                threads_use
+            } else {
+                debug!("Could not get threads use.");
+                -1_f64
+            };
 
             let mut interval = interval(Duration::from_millis(1));
```

Comment:
```suggestion
            let mut interval = interval(Duration::from_millis(1));
```

Turn the number into a constant, and let's see if increasing it benefits us? So we don't fire up every ms